zbus/src/connection/mod.rs

```diff
@@ -1217,6 +1236,56 @@ impl Connection {
             .map_err(Into::into)
     }
 
+    /// Gracefully close the connection, waiting for all other references to be dropped.
+    ///
+    /// This will not disrupt any incoming or outgoing method calls, and will await their
+    /// completion.
+    ///
+    /// # Caveats
+    ///
+    /// * Calling this will not prevent new incoming messages from keeping the collection alive, so
+    ///   you may want to take precautions like deregistering well-known names to reduce the
+    ///   likelihood of this occurring.
```

Comment:
As I [explained more now](https://github.com/dbus2/zbus/pull/921#discussion_r1716900578), I think we should remove the part after `alive` here.

Reply:
Done